### PR TITLE
improve codebase: add unit testing

### LINKED ISSUE #176: Unit Testing
Deciding if we need to setup GitHub Actions Runner Unit testing in a Docker instance because the set price feature was broken due to the labels being changed. These sort of capabilities must be functional!

We should set up unit tests.

### PR DESCRIPTION
**Pull request by molecula451**
Resolves #176

### CODE REVIEW

**Review by pavlovcik — APPROVED** [review]
This seems like a pretty solid start. Thanks for your high quality first time contribution to this repository @molecula451 !

**Review by pavlovcik — COMMENTED** [review]
I reviewed my specification and I still think we should consider setting up a GitHub Actions Docker runtime. @rndquu @molecula451 curious on your input here.

**Review by rndquu — CHANGES_REQUESTED** [review]
Pls add more [tests](https://github.com/ubiquity/ubiquibot/issues/176#issuecomment-1637534000) along with CI test workflow

**Comment by molecula451** [comment]
> 
> We [plan](https://github.com/ubiquity/ubiquibot/issues/176#issuecomment-1637534000) to go step by step:
> 
> 1. Unit tests
> 2. Functional tests (perhaps using docker)

CI already working and testing on https://github.com/ubiquity/ubiquibot/actions/runs/5574793016/jobs/10183863908

Adding and polishing more tests

**Comment by molecula451** [comment]
> I reviewed my specification and I still think we should consider setting up a GitHub Actions Docker runtime. I haven't done a deep dive on researching these matters so I'm not 100% confident on the best way forward. The business problem I am aiming to solve is to minimize or eliminate the reliance on our manual quality assurance processes.
> 
> The worst is when pull requests pass reviews, and then bugs appear in staging/production.
> 
> @rndquu @molecula451 curious on your input here.

and I am in pair with that vision, but I remind that's why you have @rndquu and **I** for the good QA and making sure things work as of highest standard

File: src/tests/help.test.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { generateHelpMenu } from "../handlers/comment/handlers/index";
+
+jest.mock("../handlers");
+jest.mock("../configs");
+
+describe("Help", () => {
+  it("should generate log menu", () => {
+    const helpMenu = generateHelpMenu();
+    expect(helpMenu).toBeTruthy();
```
Comment (molecula451) [inline src/tests/help.test.ts:9]: this truthy it's better on this one as `generateHelpMenu()` is prone to change

**Comment by pavlovcik** [comment]
Stale

**Comment by molecula451** [comment]
@pavlovcik this was not at all stale thing is rndquu changed the game of the original descripción which was to create an initial base for unit testing ive opened the PR before the asked changed. That i stale a bit. But then i reincorporate and i was desigining a Payload mock testing for the commands this is not trivial and was taking time

**Comment by pavlovcik** [comment]
The last update on this was 19 days ago. It doesn't look like it's being worked on anymore which is why I closed it.

**Pull request closed without merging**